hack/lib/golang.sh

```diff
@@ -240,6 +239,7 @@ kube::golang::set_platform_envs() {
     if [[ ${platform} == "linux/arm" ]]; then
       export CGO_ENABLED=1
       export CC=arm-linux-gnueabi-gcc
+      export GOROOT=${K8S_PATCHED_GOROOT}
```

Comment:
this might cause an issue if we get into this branch not inside the build image, though I think that'd only happen if you try to build k8s while on linux/arm.